paddle/fluid/operators/math/depthwise_conv.cu

```diff
@@ -142,13 +141,14 @@ __device__ __inline__ void KernelDepthwiseConvNHWC(
     for (int w_in = w_in_start; w_in < w_in_end; w_in += dilate_width) {
       if (h_in >= h_start && h_in < h_end && w_in >= w_start && w_in < w_end) {
         int offset = ((batch * input_height + h_in) * input_width + w_in) *
-                         output_channels +
+                         input_channels +
```

Comment:
The original code here seems to cause an error when input_channels is not equal to the output_channels. We will add a case in unit tests.

Reply:
Yes, it should be `input_channels` here.